packages/core/test/merkle-distributor/MerkleDistributor.js

```diff
@@ -615,7 +624,22 @@ contract("MerkleDistributor.js", function(accounts) {
         count++;
       }
       const average = total.divn(count);
-      assert.equal(Math.floor(average.toNumber()), 84831);
+      assert.equal(Math.floor(average.toNumber()), 84809);
+    });
+    it("(batch claim): gas average random distribution", async function() {
+      for (let i = 0; i < NUM_LEAVES; i += NUM_LEAVES / SAMPLE_SIZE) {
+        const leaf = rewardLeafs[i];
+        const proof = merkleTree.getProof(leaf.leaf);
+        batchedClaims.push({
+          windowIndex: windowIndex,
+          account: leaf.account,
+          accountIndex: leaf.accountIndex,
+          amount: leaf.amount,
+          merkleProof: proof
+        });
+      }
+      const tx = await merkleDistributor.claimMulti(batchedClaims, [rando], [rewardToken.address]);
+      assert.equal(Math.floor(tx.receipt.gasUsed / batchedClaims.length), 49304);
```

Comment:
Compare to 56846 gas for #2672 [here](https://github.com/UMAprotocol/protocol/pull/2672/files#r590803080)